src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -96,6 +94,38 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return filter.positiveAttributeFilter ? false : true;
```

Comment:
This will not work with typed filter.
AFM.isPositiveAttributeFilter(filter)

Reply:
Done.